ui/webui/Makefile.am

```diff
@@ -89,6 +89,7 @@ $(COCKPIT_REPO_STAMP): Makefile.am
 bots: tools/make-bots
 	GITHUB_BASE="cockpit-project/cockpit" tools/make-bots
 	cd test && ln -s ../bots/images images
+	cd bots/machine/machine_core && cp machine_virtual.py machine_virtual_original.py && ln -sfT ../../../test/machine_virtual.py machine_virtual.py
```

Comment:
I wonder what is blocking us from using the `test/common/run-tests` directly?

Reply:
I'm working on making `test/common/run-tests` pick a correct `machine_class`

https://github.com/cockpit-project/cockpit/pull/18627

This would require anaconda's test classes to hardcode the machine_class which this PR removes. 

```
from machine_install import VirtInstallMachine

@nondestructive
class TestReview(MachineCase):
    MachineCase.machine_class = VirtInstallMachine
```

Honestly this should probably be an anaconda specific class so we don't have to copy paste MachinceCase.machine_Class = VirtInstallMachine around.